yjit/src/core.rs

```diff
@@ -1561,7 +2149,7 @@ unsafe fn add_block_version(blockref: BlockRef, cb: &CodeBlock) {
     let block = unsafe { blockref.as_ref() };
 
     // Function entry blocks must have stack size 0
-    assert!(!(block.iseq_range.start == 0 && block.ctx.stack_size > 0));
+    //assert!(!(block.iseq_range.start == 0 && block.ctx.stack_size > 0));
```

Comment:
Is it possible to keep the assertion?
```suggestion
    assert!(!(block.iseq_range.start == 0 && block_ctx.stack_size > 0));
```

Reply:
I commented it because executing the assertion will require decoding the context, but I could turn it into a `debug_assert` ?

Reply:
Oh sorry, I thought L2052 was in the same function but it wasn't. Then it makes sense. But yeah, making it a `debug_assert` would be nice too.